crates/ruma-state-res/.clippy.toml

```diff
@@ -0,0 +1,10 @@
+msrv = "1.50"
+disallowed-types = []
```

Comment:
This looks like a no-op, is this actually required? I thought Clippy doesn't do config inheritance.

Reply:
It's not I just left it as a sorta reminder to add if someone could think of any? I'll get rid of it?

Reply:
That's probably a good idea to have it as a reminder.